tests/unit/components/plate_based_widgets/mapeditor/PlateMapEditor.spec.js

```diff
@@ -376,8 +174,7 @@ describe("PlateMapEditor.vue", () => {
         expect(well1.attributes("stroke-width")).toBe("0");
       }
 
-      const buttonclicked = wrapper.find(string_name_of_button);
-      await buttonclicked.trigger("click");
+      await await wrapper.vm.basic_select(well_indices);
```

Comment:
what effect does having two `await` commands have? I haven't seen that before.

Is there a reason you changed from `trigger('click')` to `vm.basic_select`?  Typically we want to try and mimic user interactions (such as a click) rather than directly invoking methods inside the component

Reply:
Modified all them to use any of the following:-
a) trigger('click')
b) trigger('click' { shiftKey : true })
c) trigger('click' { ctrlKey : true })

And this now emits and vue event 'click-exact', 'click-shift-exact' and 'click-ctrl-exact' so this will result in the respective bounded functions been invoked.